client/my-sites/checkout/controller.jsx

```diff
@@ -52,8 +52,8 @@ export function checkout( context, next ) {
 
 	context.store.dispatch( setSection( { name: 'checkout' }, { hasSidebar: false } ) );
 
-	if ( config.isEnabled( 'composite-checkout' ) ) {
-		context.primary = <CompositeCheckoutContainer />;
+	if ( config.isEnabled( 'composite-checkout-wpcom' ) ) {
```

Comment:
Let's just remove all the changes in this file and rename your work to `CompositeCheckoutContainer`.